### ISSUE TITLE
[all] page titles

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag-redesign#16]
* [ ] **all** &mdash; to end of all titles, add: 
```
WCAG | WAI | W3C
```
(like [ACT Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/about/) and others)
* [ ] **Understanding docs** &mdash; Delete "Success Criterion" in "Understanding Success Criterion 1.1.1: Non-text Content" so it's "Understanding 1.1.1: Non-text Content"
* [x] _leave Guideline in "Understanding Guideline 1.1: Text Alternatives"_
* [ ] **Techniques** &mdash; Delete the full technique phrase from title. Just have "Technique" and the ID,
   * ideally with the technology/category written out, e.g.,:
   > Technique General 34 | WCAG | WAI | W3C
   > Technique HTML 56 | WCAG | WAI | W3C
   > Technique Failure 78 | WCAG | WAI | W3C
   * OK if only the short bit for now, e.g.,:
   > Technique G34 | WCAG | WAI | W3C
   > Technique H56 | WCAG | WAI | W3C
   > Technique F78 | WCAG | WAI | W3C

**Comment by michael-n-cooper** [comment]
This should be straightforward but I need the specific choice for the technique titles, not a list of options. They are of equivalent difficulty.

**Comment by shawna-slh** [comment]
updated as requested

**Comment by shawna-slh** [comment]
created issue in main WCAG repo
